previewer/metadata_renderer.js

```diff
@@ -12,6 +12,8 @@ const visibleProps = [ 'assetID',
                        'matchingLinks',
 
                        'title',
+                       'synopsis',
+                       'thumbnail',
```

Comment:
Thumbnails should be good to include in the asset listing on the left pane as well (on the left side of the asset name) when previewing.